modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/display/context/DDMDisplayContext.java

```diff
@@ -309,15 +309,9 @@ public String getRefererPortletName() {
 	}
 
 	public String getScopedStructureLabel() {
-		String scopeTitle = ParamUtil.getString(_renderRequest, "scopeTitle");
```

Comment:
@rodrigopaulino, can you please give me more details why this is not necessary any more?

If I've understood correctly, this commit will remove the parameter `scopeTitle` from the URL, am I right? If so, won't it generate any side effect?

Should we use something like `HtmlUtil.extractText(ParamUtil.getString(_renderRequest, "scopeTitle"));` instead?

Reply:
I sure can give you more details, @natocesarrego.

Basically, this was giving access to anyone to change the name of the scope of the portlet. Meaning, if I wanted to change the message "There are no Data Definitions" that appears on the search container list to "There are no stuff", it would be possible by editing the value of this parameter. I figured we shouldn't allow that. Escaping, or extracting the text would only get rid of the possibility of script injections. Does it make sense to you? :)

Please let me know if you need further details. 
Thanks for reviewing this.

Reply:
Makes sense, @rodrigopaulino. :slightly_smiling_face: 

So, your point is that the `scopeTitle` parameter isn't necessary, right?

In light of this, do you think that the absence of this parameter will generate any side effect?

Thanks. :wink:

Reply:
I don't think so, @natocesarrego. I'll be happy to change it if you see any. 😃